### DISCUSSION TITLE
Using Typer's custom CSS/JS for Termynal

### DISCUSSION THREAD

**Discussion by NikosAlexandris** [fastapi/typer#712] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the Typer documentation, with the integrated search.
- [X] I already searched in Google "How to X in Typer" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to Typer but to [Click](https://github.com/pallets/click).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
# No example code needed!
```

### Description

**Is it OK to borrow** Typer's custom [CSS](https://github.com/tiangolo/typer/tree/master/docs/css)/[JS](https://github.com/tiangolo/typer/tree/master/docs/js) files for integrating Termynal in my own (still private) project **?**

### Operating System

Linux

### Typer Version

Not important!

### Python Version

Not important!

**Comment by YuriiMotov** [comment, marked as answer]
MIT license allows commercial use and private use.

See license: https://github.com/fastapi/typer/blob/master/LICENSE
